_component_design/choice-tiles.md

```diff
@@ -0,0 +1,224 @@
+usage: >
+  Choice tiles are an alternative selector pattern to radio buttons and checkboxes that provide a means of both displaying choices more visually and of providing a more detailed description of a choice.
+preview-image: preview-images/choice-tile.svg
+status: stable
+resource: true
+last-modified: 2017-11-12
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### When to use
+
+- Use choices tiles when presenting a set of choices that are easily recognized by iconography.
+- Only use choice tiles when two or more selections are available.
+
+### Best practices
+
+- When using a description keep the text brief to ensure users engage with are not overwhelmed by a wall across multiple choice tiles.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The choice tile pattern"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-hero.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Default choice tile
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Cards are composed of the following:
+
+- **Choice tile**: the container that houses the choice tile selection.
+- **Icon**: an iconographic representation of the selection contained within a choice tile.
+- **Heading**: the title of the selection contained within a choice tile.
+- **Description** *(optional)*: a description assists in briefly explaining the choice tile selection to a user.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Standard layout for a choice tile."] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-default.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+### Layout options
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- Use a choice tile to capture a user selection within a form.
+- Choice tiles comes in small, medium, and large sizes. Only a singular size may be used when making a selection choice. 
+- Choice tile options **do not** exceed nine options.
+- Small tiles **do not** span more than 6 columns across (max 6 tiles in a row).
+- Medium tiles **do not** span more than 8 columns across (max 4 tiles in a row).
+- Large tiles **do not** span more than 9 columns across (max 3 tiles in a row).
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Choice tiles on a full-width grid with no right sidebar"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-layout.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+### Responsive design specifications
```

Comment:
Change to `Responsive Specifications` for brevity

Reply:
updated. Responsive specifications

Reply:
also moved into the column left